heudiconv/utils.py

```diff
@@ -265,6 +265,32 @@ def json_dumps_pretty(j, indent=2, sort_keys=True):
     return js_
 
 
+def add_field_to_json(json_file, field_and_value):
```

Comment:
```suggestion
def update_json(json_file, data):
```
or alike would be a better name:
- `.update` might not `add` but replace
- it is not just one but many (entire dict).

note: may be in the future we would like to extend it to use https://pypi.org/project/deepmerge/ ?

Reply:
Done in 0650b68a2c6a936915ec96855993a935c103d4b4